microscopium/bokeh_app.py

```diff
@@ -169,7 +180,7 @@ def load_selected(attr, old, new):
 
         glyphs.data_source.on_change('selected', load_selected)
 
-        fig = row([pca, sel], sizing_mode='stretch_both')
+        fig = row([pca, sel], sizing_mode="stretch_both")
```

Comment:
This change seems unnecessary?

Reply:
Sorry - I lost the sizing mode keyword in that line for a bit while I was mucking around with the aspect ratio keyword arguments. There's not supposed to be a change here.